x-translator/smartapi_x-translator_schema.json

```diff
@@ -53,5 +59,63 @@
         }
       }
     }
+  },
+  "allOf":[
+    {
+      "if": {
+        "required": ["info"],
+        "properties": {
+          "info": {
+            "required": ["x-translator"]
+          }
+        }
+      },
+      "then": {
+        "required": ["tags"],
+        "properties": {
+          "tags": {
+            "contains": {
+              "type": "object",
+              "required": ["name"],
+              "properties":
+              {"name": {"const": "translator"}}
+            }
+          }
+        }
+      }
+    },
+    {
+      "if": {
+        "required": ["tags"],
+        "properties": {
+          "tags": {
+            "contains": {
+              "type": "object",
+              "required": ["name"],
+              "properties":
+              {"name": {"const": "translator"}}
+            }
+          }
+        }
+      },
+      "then": {
+        "required": ["info"],
+        "properties": {
+          "info": {
+            "required": ["x-translator"]
+          }
+        }
+      }
+    }
+  ], 
+  "definitions": {
+    "tag": {
+      "properties": {
+        "x-id": {
+          "type": "string",
+          "description": "The name of the tag. Recommend that you use URI to specify the concept."
+        }
```

Comment:
why this part is needed?

Reply:
@newgene This was copied from the SmartAPI schema [here](https://raw.githubusercontent.com/SmartAPI/smartAPI-Specification/OpenAPI.next/schemas/smartapi_schema.json). 

In the new commit, I have updated the tags sub-schema. I believe it now reflects the way tags are currently used (a list of objects, each object has the key "name" and then a string value).